index.html

```diff
@@ -201,12 +201,65 @@ <h3 class="tonic flex0">Uber Navigation</h3>
             <li class="highlight font-2">css</li>
             <li class="highlight font-3">javaScript</li>
           </ul>
-          <a href="project2.com" class="highlight-2 font-4 flex0">
+          <button class="highlight-2 font-4 flex0 btns">
             See Project
-          </a>
+          </button>
         </div>
       </div>
     </section>
+
+    <!-- Pop Up Window -->
+    <div class="bg-modal">
+      <!-- <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="tonic">Tonic</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class="ul-box1 flex0">
+          <li class="canopy">CANOPY</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">Back End Dev</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">2015</li>
+        </ul>
+        <a href="#"
+          ><img src="images/c1d.png" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="flex-99">
+          <div class="flex-98">
+            <p class="description-2 flex0">
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s.
+              <span class="description-3">
+                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt
+                animi consequuntur consectetur voluptate accusantium facilis,
+                fuga non minus,
+              </span>
+            </p>
+          </div>
+          <div>
+            <ul class="ul-box2 flex0">
+              <li class="highlight font-1">html</li>
+              <li class="highlight font-2">css</li>
+              <li class="highlight font-3">javaScript</li>
+            </ul>
+            <hr class="solid-1">
+            <div class="buttons">
+              <button type="button" class="live-btn">
+                see live
+                <i class="fas fa-check"></i>
+              </button>
+              <button type="button" class="live-btn">
+                see source
+                <i class="fab fa-github"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div> -->
```

Comment:
[ ] As asked in the last point in[ HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) kindly remove the commented code

![image](https://user-images.githubusercontent.com/73140461/151216189-94aae582-df40-405e-abf6-43b81645d755.png)